code/pizzabot.js

```diff
@@ -1,7 +1,140 @@
+//Variables defining data
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
 
-const pizzaPrice = 80
+const pizzaPrice = 80;
+
+const vegPic = document.getElementById("vegPizza");
+const pepPic = document.getElementById("pepPizza");
+const hawPic = document.getElementById("hawPizza");
+
+//--------------------------------------------------------------------------//
+
+//Sentances showing in html via the getElementById()
+// Greeting to the user from the pizzabot
+document.getElementById("greeting").innerHTML = `&#129302 Hi! Welcome to Claire & Claudia's pizza ordering service. I'm pizza bot and I'm here to help you with your order.`;
+
+//Showing the menu
+document.getElementById("menu").innerHTML = `On the menu we have: <li>${vegetarian}: ${pizzaPrice}kr</li> <li>${hawaiian}: ${pizzaPrice}kr</li> <li>${pepperoni}: ${pizzaPrice}kr</li>`;
+
+//Message to the user asking them to choose a pizza from the menu
+document.getElementById("choosePizza").innerHTML = "&#129302 Ok great. Let's get started with your order! Please write the name of the pizza you want to order:";
+
+//--------------------------------------------------------------------------//
+
+//Function compares if the parameter chosenPizza is true or false. Will return true if user inputs correct pizza name, otherwise will show message which is called further down in enteredPizzaValue
+const validateOrderName = (chosenPizza) => {
+    if (chosenPizza.toLowerCase() === vegetarian.toLowerCase() || chosenPizza.toLowerCase() === hawaiian.toLowerCase() || chosenPizza.toLowerCase() === pepperoni.toLowerCase()){
+        return true;
+    }
+    else { 
+        return false;
+    }
+};
+
+//Function calculating the cost of the pizza based on the pizzaAmount and pizzaPrice. Returnsg the amount when function is called further down
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+    const orderTotal = (orderQuantity * pizzaPrice);
+    return orderTotal;     
+};
+
+//Function calculating the cooking time based on amount of pizzas ordered
+const calculateCookingTime = (pizzaAmount) => {
+    if (pizzaAmount <= 2) {
+        return 10;
+    } else if (pizzaAmount >= 3 && pizzaAmount <= 5) {
+        return 15;
+    } else {
+        return 20;
+    }
```

Comment:
Alternative logic:
```javascript
    if (pizzaAmount <= 2) {
        return 10;
    } else if (pizzaAmount >= 6) {
        return 20;
    } else {
        return 15;
    }
```

The code does the same thing, although this isn't necessarily better or worse, the trade-offs are:
- The return values aren't ascending when you read them (10, 20, 15)
- There is only one conditional in the `else if`